src/features/auth/components/Header/index.js

```diff
@@ -2,22 +2,48 @@ import React from 'react';
 import { Trans } from 'react-i18next';
 import { Link } from 'react-router-dom';
 
-import { Header, Menu } from '../../../../common/components';
+import { Header, Icon, Menu, Popover } from '../../../../common/components';
 
 const AuthHeader = () => (
   <Header>
-    {() => [
-      <Menu.Item key="signup">
-        <Link to="/auth/sign-up">
-          <Trans i18nKey="nav.signup">Sign Up</Trans>
-        </Link>
-      </Menu.Item>,
-      <Menu.Item key="login">
-        <Link to="/auth/login">
-          <Trans i18nKey="nav.login">Log In</Trans>
-        </Link>
-      </Menu.Item>,
-    ]}
+    {({ activePathname, hide, show, isVisible, isMobile }) => {
```

Comment:
lets rename activePathname to something menu specific

Reply:
also isVisible, hide and show should be more clear, currently i have no idea what it represents

Reply:
ad renaming `activePathname` first of all, shouldn't we name the component ResponsiveNavMenu as it's only for navigation because it works with router history.

Next, I wouldn't obfuscate the actual value of the prop. The activePathname says what it is. Could be
- active/current/selected + route/location/pathname/item/routeKey?

all seems fishy.

Ad prop naming: I was of the opinion of keeping the props passed to render prop (often when consuming context) in a raw, hollow fashion, because it's clear that it ties to the consumer. e.g. 
```
<Menu>{({hide, show, visible}) =>
  visible ? <Button onClick={hide} /> : <Button onClick={show} />}
</Menu>
```

vs
```
<Menu>{({hideMenu, showMenu, isMenuVisible}) =>
  visible ? <Button onClick={hideMenu} /> : <Button onClick={showMenu} />}
</Menu>
```